src/net.h

```diff
@@ -523,10 +541,16 @@ class CNode
         return m_conn_type == ConnectionType::INBOUND;
     }
 
+    bool IsManualBlockRelayConn() const
```

Comment:
Since you're adding `IsManualBlockRelayConn()`, consider renaming `IsBlockOnlyConn()` to `IsAutomaticBlockRelayConn()`, otherwise that function name is misleading (it returns `false` for connections that are block relay only connections but were created manually).

Reply:
Done, see also https://github.com/bitcoin/bitcoin/pull/24170#discussion_r863023967.